jdr/jboss-as-jdr/src/main/java/org/jboss/as/jdr/JdrRunner.java

```diff
@@ -108,6 +108,7 @@ public JdrReport collect() throws OperationFailedException {
             }
             versionWriter.close();
             this.env.getZip().add(new ByteArrayInputStream(versionStream.toByteArray()), "version.txt");
+            this.env.getZip().add(new ByteArrayInputStream(this.env.getZip().getProductDirName().getBytes(StandardCharsets.UTF_8)), "product.txt");
```

Comment:
@bmaxwell Any objections to this? It's adding a file to the JDR zip.

Reply:
No issues